resources/assets/less/app.less

```diff
@@ -29,6 +29,9 @@
 @import (inline) "@{node_root}/photoswipe/dist/photoswipe.css";
 @import "photoswipe-fix";
 
+@import (inline) "@{node_root}/react-table/react-table.css";
+@import "react-table-overrides.less";
```

Comment:
extension not needed? Also I noticed at the other place it's suffixed with `-fix` =D (see above)